blueprints/jsonapi-adapter/files/__root__/__path__/__name__.js

```diff
@@ -1,17 +1,5 @@
 <%= importStatement %>
-import config from '../config/environment';
 
 export default <%= baseClass %>.extend({
   type: '<%= entity %>',
-
-  url: config.APP.API_PATH + '/<%= resource %>',
```

Comment:
I like the generated `resource` name and use of `fetchUrl` to customize as needed, see https://github.com/pixelhandler/ember-jsonapi-resources/blob/master/addon/adapters/application.js#L337-L347